web/src/components/home/Testimonial.vue

```diff
@@ -34,7 +34,7 @@
                 <div class="media">
                   <div class="media-left">
                     <figure class="image is-48x48">
-                      <img class="is-rounded" src="../../assets/bedford.webp" alt="Jesper" />
+                      <img class="is-rounded" src="../../assets/bedford.jpeg" alt="Jesper" />
```

Comment:
```suggestion
                      <img class="is-rounded" src="../../assets/bedford.jpeg" alt="Bedford" />
```

I accidentally made this typo when writing that part. Sorry!

Reply:
@tbaumer22 No problem at all! Updated!